### PR TITLE
feat: namespaced operator

### PR DESCRIPTION
**Pull request by adityathebe**
resolves: https://github.com/flanksource/canary-checker/issues/2036

resolves: https://github.com/flanksource/canary-checker/issues/2041

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -44,6 +44,17 @@ jsonLogs: true
 # restrict canary-checker to monitor single namespace for canaries.  Leave blank to monitor all namespaces
 canaryNamespace: ""
 
+# Only canaries matching these filters will run.
+# If left empty, all canaries will run.
+canaryFilter:
```
Comment (moshloop) [inline chart/values.yaml:49]: can we use canaryNamespaceSelector and canarySelector to be more consistent with prometheus

File: chart/templates/rbac.yaml [hunk 2]
```diff
@@ -145,3 +146,17 @@ rules:
       - get
       - update
       - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (moshloop) [inline chart/templates/rbac.yaml:150]: Why do we need this ?

Comment (adityathebe) [reply chart/templates/rbac.yaml:150]: don't we need the binding?

This is coming from `chart/templates/clusterrolebinding.yaml` which got deleted and merged here.

**Comment by nalshamaajc** [comment]
Thanks for the prompt response, @moshloop when will these changes be released in a chart version?

**Comment by adityathebe** [comment]
@nalshamaajc this should be available on the latest release.

**Comment by nalshamaajc** [comment]
Thank you @adityathebe, when will the next stable release be (v1.0.260)?

**Comment by adityathebe** [comment]
> Thank you @adityathebe, when will the next stable release be (v1.0.260)?

@nalshamaajc we have automatic beta releases that you can test.

And we're a week or 2 away from `v1.0.261`

**Pull request merged**